terraform/dns.tf

```diff
@@ -186,6 +186,11 @@ locals {
       type     = "TXT"
       value    = "v=spf1 include:spf.improvmx.com ~all"
     },
+    {
+      hostname = "live.nixos.org"
+      type     = "A"
```

Comment:
Where is IPv6?

Reply:
It currently has none. Will add one if this is necessary.